### PR TITLE
[NEURON][BUILD] - PT1.10, TF2.5 to sdk1.19.0 and mx1.8 to sdk1.18.0

### PR DESCRIPTION
**Pull request by aws-vrnatham**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Move to neuron sdk1.9.0 for pt,tf and sdk1.8.0 for mx18.

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pytorch/buildspec-1-10-1-neuron.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK pytorch
-version: &VERSION 1.10.1
+version: &VERSION 1.10.2
```
Comment (saimidu) [inline pytorch/buildspec-1-10-1-neuron.yml:4]: Please also change the name of this file to `buildspec-1-10-2-neuron.yml.

File: tensorflow/inference/docker/2.5/py3/sdk1.19.0/Dockerfile.neuron [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    requests==2.25.1 \
+    grpcio==1.34.1 \
+    protobuf==3.14.0 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api==2.5.3
+
+# pip install statements have been separated out into multiple sequentially executed statements to
+# resolve package dependencies during installation.
+RUN ${PIP} install "tensorflow_serving_api==2.5.3" --no-deps --extra-index-url https://pip.repos.neuron.amazonaws.com \
```
Comment (saimidu) [inline tensorflow/inference/docker/2.5/py3/sdk1.19.0/Dockerfile.neuron:105]: Is there a custom tensorflow-serving-api on the neuron pip repo? I don't see it on the webpage when I visit it.

**Review by saimidu — COMMENTED** [review]
LGTM, apart from the 3 comments.

**Review by saimidu — APPROVED** [review]

File: tensorflow/inference/docker/2.5/py3/sdk1.19.0/Dockerfile.neuron (hunk 2 above)
Comment (aws-vrnatham) [reply tensorflow/inference/docker/2.5/py3/sdk1.19.0/Dockerfile.neuron]: Took care of removing the extra install and also the extra index.

File: pytorch/buildspec-1-10-1-neuron.yml (hunk 1 above)
Comment (aws-vrnatham) [reply pytorch/buildspec-1-10-1-neuron.yml:4]: Done

File: mxnet/inference/docker/1.8/py3/sdk1.18.0/Dockerfile.neuron [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.1.8
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.10
```
[the remark this answers is not in the document]
Comment (aws-vrnatham) [reply mxnet/inference/docker/1.8/py3/sdk1.18.0/Dockerfile.neuron:17]: Since just updating the neuron sdk version don't want to update the python version. In the new PT 1.11 container release will try this new python version and fix all the issues that are found with using that python version in the test/install failures.

**Review by saimidu — APPROVED** [review]
